src/integration/sync_integration_test.go

```diff
@@ -22,6 +22,8 @@ func TestSocketConnection(t *testing.T) {
 
 	testcases := []struct {
```

Comment:
Make this test hit the write and delete endpoints instead of the publish endpoint.  That way we know everything works end to end.